<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exam</class>
 <widget class="QMainWindow" name="exam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ASL Learning - Examination</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>favasl.png</normaloff>favasl.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Question No.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_qno">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;01&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_time">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;time&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_q">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>610</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_skip">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>610</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Skip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_nxt">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>610</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Submit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>610</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_date">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;date&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_snap">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>610</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Snap</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cam">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
